src/arrow-sdk.ts

```diff
@@ -1,248 +1,265 @@
+  },
+  provider: {
+    fuji: "https://api.avax-test.network/ext/bc/C/rpc",
+  },
+};
 
 export const providers: any = {
-    "fuji": new ethers.providers.JsonRpcProvider(urls.provider.fuji)
-}
+  fuji: new ethers.providers.JsonRpcProvider(urls.provider.fuji),
+};
 
 export const addresses: any = {
-    "fuji": {
-        "router": {
-            [VERSION.V2]: ethers.utils.getAddress("0x28121fb95692a9be3fb1c6891ffee74b88bdfb2b"),
-            [VERSION.V3]: ethers.utils.getAddress("0x31122CeF9891Ef661C99352266FA0FF0079a0e06"),
-            [VERSION.COMPETITION]: ethers.utils.getAddress("0x3e8a9Ad1336eF8007A416383daD084ef52E8DA86")
-        }
-    }
-}
+  fuji: {
+    router: {
+      [VERSION.V2]: ethers.utils.getAddress(
+        "0x28121fb95692a9be3fb1c6891ffee74b88bdfb2b"
+      ),
+      [VERSION.V3]: ethers.utils.getAddress(
+        "0x31122CeF9891Ef661C99352266FA0FF0079a0e06"
+      ),
+      [VERSION.COMPETITION]: ethers.utils.getAddress(
+        "0x3e8a9Ad1336eF8007A416383daD084ef52E8DA86"
+      ),
+    },
+  },
+};
 
 export const bytecodeHashes: any = {
-    "ArrowOptionChainProxy": {
-        [VERSION.V2]: ethers.utils.solidityKeccak256(
-            ['bytes'],
-            [ArrowOptionChainProxy.v2.bytecode]
-        ),
-        [VERSION.V3]: ethers.utils.solidityKeccak256(
-            ['bytes'],
-            [ArrowOptionChainProxy.v3.bytecode]
-        ),
-        [VERSION.COMPETITION]: ethers.utils.solidityKeccak256(
-            ['bytes'],
-            [ArrowOptionChainProxy.competition.bytecode]
-        )
-    }
-}
+  ArrowOptionChainProxy: {
+    [VERSION.V2]: ethers.utils.solidityKeccak256(
+      ["bytes"],
+      [ArrowOptionChainProxy.v2.bytecode]
+    ),
+    [VERSION.V3]: ethers.utils.solidityKeccak256(
+      ["bytes"],
+      [ArrowOptionChainProxy.v3.bytecode]
+    ),
+    [VERSION.COMPETITION]: ethers.utils.solidityKeccak256(
+      ["bytes"],
+      [ArrowOptionChainProxy.competition.bytecode]
+    ),
+  },
+};
 
 /***************************************
  *           ARROW API CALLS           *
  ***************************************/
 
 /**
  * Get an estimated price for the given option parameters from Arrow's pricing model.
- * 
+ *
  * @param option Object containing parameters that define an option on Arrow.
  * @param version Version of Arrow contract suite with which to interact. Default is V2.
  * @returns Float that represents an estimate of the option price using Arrow's pricing model.
  */
-export async function estimateOptionPrice(option: Option, version: VERSION = VERSION.V2) {
-    let strike = undefined
-    switch(version) {
-        case VERSION.V2: 
-            strike = option.strike
-            break
-        case VERSION.V3:
-        case VERSION.COMPETITION: 
-            strike = (option.strike as number[]).join('|')    
-            break
-        default:
-            throw UNSUPPORTED_VERSION_ERROR
+export async function estimateOptionPrice(
```

Comment:
This is the functional change. Everything else is eslint cleaning up and standardizing the code/